Lib/poplib.py

```diff
@@ -226,8 +226,16 @@ def stat(self):
         retval = self._shortcmd('STAT')
         rets = retval.split()
         if self._debugging: print('*stat*', repr(rets))
-        numMessages = int(rets[1])
-        sizeMessages = int(rets[2])
+
+        if len(rets) < 3:
```

Comment:
I originally thought "why not just require exactly 3 fields"? I would add a comment here that RFC 1939 says that there can be additional data past the size field. I'm assuming 1939 hasn't been superceded, but I haven't really looked into it.

Reply:
Thanks @ericvsmith, I've added the comment for RFC 1939 says that there can be additional data past the size field